content/webapp/views/concepts/concept/concept.ImagesResults.tsx

```diff
@@ -132,7 +136,7 @@ const ImagesResults: FunctionComponent<{
 
   return (
     <>
-      <ThemeImagesWrapper as="section" data-testid="images-section">
+      <ThemeImagesWrapper data-testid="images-section">
```

Comment:
This reminds me that we probably want to run the e2es with the concept toggles turned on; we're likely to have flags come up.